### DISCUSSION TITLE
Некорректная генерация аудио

### DISCUSSION THREAD

**Discussion by Khimer** [snakers4/silero-models#143] [Q&A] [answered]
За основу взят код https://colab.research.google.com/github/snakers4/silero-models/blob/master/examples_tts.ipynb#scrollTo=31a041ac
1. v2_Aidar - пропуск части текста при генерации аудио с помощью батча (проблема сохраняется на gpu  и на cpu):
https://colab.research.google.com/drive/1DEQEI12cuIB4hhTXvJ91zGvRxziTg37s?usp=sharing
При генерации по одному файлу проблемы нет.
2. v2_baya - пропуск/зашумление слов.  При использовании батча таких ошибок больше:
https://colab.research.google.com/drive/1xNnuEzglTq_w4HtCgDJ6nVKhKe_vNCOZ?usp=sharing
У остальных спикеров подобные проблемы замечены не были.

**Comment by snakers4** [comment]
(1)
Пока решается батч сайзом 1
В будущих релизах такой проблемы не будет

(2)
Аналогично

**Comment by snakers4** [comment, marked as answer]
Решается релизом новой `V3` русской модели.
